scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl

```diff
@@ -286,8 +286,8 @@ task AssertCostIsTrackedAndExpected {
 
         echo "project_id = ~{project_id}" > ~/.bigqueryrc
         bq --apilog=false query --project_id=~{project_id} --format=csv --use_legacy_sql=false \
-            'SELECT call, step, event_key, sum(event_bytes)
-              FROM `~{dataset_name}.cost_observability`
+            'SELECT call, step, event_key, sum(event_bytes) FROM
+                (SELECT distinct step, call, event_key, event_bytes, shard_identifier FROM `~{dataset_name}.cost_observability`)
```

Comment:
The `distinct` is treating `event_bytes` like it's a component of a composite key?

Reply:
This was an error. I have rewritten the query.